Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -183,16 +187,16 @@ static void parse_special_casing(Core::File& file, UnicodeData& unicode_data)
         casing.uppercase_mapping = parse_code_point_list(segments[3]);
 
         if (auto condition = segments[4].trim_whitespace(); !condition.is_empty()) {
-            auto conditions = condition.split(' ', true);
+            auto conditions = condition.split_view(' ', true);
             VERIFY(conditions.size() == 1 || conditions.size() == 2);
 
             if (conditions.size() == 2) {
-                casing.locale = move(conditions[0]);
-                casing.condition = move(conditions[1]);
+                casing.locale = conditions[0];
+                casing.condition = conditions[1];
             } else if (all_of(conditions[0], is_ascii_lower_alpha)) {
-                casing.locale = move(conditions[0]);
+                casing.locale = conditions[0];
             } else {
-                casing.condition = move(conditions[0]);
+                casing.condition = conditions[0];
```

Comment:
These `casing` objects won't outlive the function-local buffer, right?

Reply:
They do, but the members are `String`, so the `StringView`s stored in `conditions` here are converted to `String`.

Reply:
Kind of spooky action at a distance, but if it works then it's all good :^)